src/styles/blocks/card.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  width: $card-width;
+  border: $card-border;
+  border-radius: 5px;
+  box-sizing: border-box;
+  cursor: pointer;
+  transition-duration: $transition-duration;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    background-image: url(../images/imac.jpeg);
+    height: 134px;
+    width: 160px;
+    background-size: 160px 134px;
+    margin: 32px 19px 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    color: $color-main;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0 16px 4px;
+    transition: color $transition-duration;
+  }
+
+  &__code {
+    font-weight: 400;
+    color: $color-secondary;
+    font-size: 10px;
+    line-height: 14px;
+    margin: $margin;
```

Comment:
The variable `$margin` is not defined in this file. Ensure it is defined in a global SCSS file or imported correctly.